sites/public/src/layouts/application.tsx

```diff
@@ -249,18 +249,8 @@ const Layout = (props: LayoutProps) => {
         {process.env.showNewSeedsDesigns ? (
           <SiteHeader
             title={t("nav.siteTitle")}
-            languages={languages?.map((lang) => {
-              return {
-                label: lang.label,
-                onClick: () =>
-                  void router.push(router.asPath, router.asPath, { locale: lang.prefix || "en" }),
-                active: t("config.routePrefix") === lang.prefix,
-              }
-            })}
-            links={getHeaderLinks(router, profile, signOut, addToast, {
-              applications: showApplications,
-              favorites: showFavorites,
-            })}
+            languages={[]}
+            links={[]}
```

Comment:
issue: This is unrelated to this line, but it won't let me comment on the correct line. Since all routes display the down for maintenance content can we remove all of the links in the footer? That would just be commenting out all of the links in `CustonSiteFooter.tsx`

Reply:
Done!